src/components/UserCreateRequestsPage/UserCreateRequestsPage.tsx

```diff
@@ -216,18 +216,23 @@ export const UserCreateRequestsPage = () => {
                             }
                             return router.internalUserRequestEdit(id);
                         };
+
+                        const requestType = () => {
```

Comment:
useMemo?

Reply:
useMemo cannot be called inside a callback